tests/test_logging/test_logging.py

```diff
@@ -52,3 +54,20 @@ def test_logging_when_scene_is_not_specified(tmp_path, python_version):
     ]
     _, err, exitcode = capture(command)
     assert exitcode == 0, err
+
+
+def test_error_logging(tmp_path):
+    path_error_scene = Path("tests/test_logging/basic_scenes_error.py")
+
+    command = [
+        sys.executable,
```

Comment:
there is a fixture called `python_version` which returns the same `sys.executable`, maybe use that?

Reply:
I was following the scheme I've seen in other tests, but if you'd prefer me to use the fixture (and say so) then I can

Reply:
I think for consistency using the fixture is better.